meetings/approvals/tests.py

```diff
@@ -1,12 +0,0 @@
-from django.test import TestCase
-from unittest import skip
-
-
-class TestApprovals(TestCase):
```

Comment:
I noticed that it's never referenced in the urls.py file and asked @Tav0 who said it could go.